coalib/bears/Bear.py

```diff
@@ -1,8 +1,15 @@
+import cProfile
 import inspect
+import io
```

Comment:
This file contains unused source code.

*Origin: PyUnusedCodeBear, Section: `flakes`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpgvrdg1da/coalib/bears/Bear.py
+++ b/tmp/tmpgvrdg1da/coalib/bears/Bear.py
@@ -3,7 +3,6 @@
 import io
 import itertools
 import pstats
-import sys
 import tempfile
 import traceback
 from functools import partial
```